setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import find_packages, setup
+
+import yolapy
+
+
+setup(
+    name='yolapy',
+    version=yolapy.__version__,
```

Comment:
It is useful when debugging env issues. For example: a virtualenv might extend system packages rather than replacing them. Things can get tricky if there are multiple versions installed (one at the system level, one at the virtualenv level). Exposing version numbers in clients lets you figure out what the hell is actually going on via a shell.